### PR TITLE
Fix case-sensitive spelling in non-common /eng

### PR DESCRIPTION
**Pull request by antkmsft**
For eng/common, see https://github.com/Azure/azure-sdk-tools/pull/7665

### CODE REVIEW

File: eng/pipelines/templates/stages/archetype-cpp-release.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#cspell: words gearama
```
Comment (LarryOsterman) [inline eng/pipelines/templates/stages/archetype-cpp-release.yml:1]: Why isn't gearama in the global cspell dictionary, along with the other team members?

Comment (LarryOsterman) [reply eng/pipelines/templates/stages/archetype-cpp-release.yml:1]: Alternative version of the above: Why are ahsonkan, Osterman, and antkmsft in the global cspell dictionary?

File: eng/scripts/Set-VcpkgWriteModeCache.ps1 [hunk 2]
```diff
@@ -1,3 +1,5 @@
+#cspell: ignore azblob
```
Comment (LarryOsterman) [inline eng/scripts/Set-VcpkgWriteModeCache.ps1:1]: Since azblob shows up several times, would it be better to put it inthe global dictionary?

**Review by LarryOsterman — APPROVED** [review]

File: eng/pipelines/templates/stages/archetype-cpp-release.yml (hunk 1 above)
Comment (antkmsft) [reply eng/pipelines/templates/stages/archetype-cpp-release.yml:1]: Thank you! Makes sense. Done.

File: eng/scripts/Set-VcpkgWriteModeCache.ps1 (hunk 2 above)
Comment (antkmsft) [reply eng/scripts/Set-VcpkgWriteModeCache.ps1]: Thank you! Makes sense. Done.

File: eng/pipelines/templates/stages/archetype-cpp-release.yml (hunk 1 above)
Comment (ahsonkhan) [inline eng/pipelines/templates/stages/archetype-cpp-release.yml:1]: We shouldn't need this anymore, correct? Because it is in the global list now?

Comment (antkmsft) [reply eng/pipelines/templates/stages/archetype-cpp-release.yml]: Thanks! Missed that one while updating. Fixed now.

**Pull request merged**